.travis.yml

```diff
@@ -19,9 +24,14 @@ gemfile:
 matrix:
   fast_finish: true
   allow_failures:
-    - rvm: ruby-head
+    - env: RB=ruby-head LIBSSL=1.0
+    - env: RB=ruby-head LIBSSL=1.1
```

Comment:
After trying hundred times, I finally make it so that Travis ignores all the builds that have the environment variable `ruby-head`. 
It's important to note that Travis won't work if you don't define the entry in the `allow_failures` matrix exactly as you define it above – see [this stackoverflow post](https://stackoverflow.com/questions/39563597/travis-ci-allow-failures-when-multiple-environment-variables-are-set). For example, this wouldn't work for me:

```yml
- env:
  - RB=ruby-head LIBSSL=1.0
  - RB=ruby-head LIBSSL=1.1

...

matrix:
  fast_finish: true
  allow_failures:
    - env: RB=ruby-head
```
Furthermore, Travis would not recognize your allowed failures on multiple environment variables entries if you define them as entries inside the same key, for example, this would also not work for me:

```yml
- env:
  - RB=ruby-head LIBSSL=1.0
  - RB=ruby-head LIBSSL=1.1

...

matrix:
  fast_finish: true
  allow_failures:
    - env: 
       - RB=ruby-head LIBSSL=1.0
       - RB=ruby-head LIBSSL=1.1
```

Reply:
Gotcha :+1: